superset-frontend/cypress-base/cypress/integration/dashboard/editmode.test.ts

```diff
@@ -535,12 +535,6 @@ describe('Dashboard edit', () => {
       applyChanges();
       saveChanges();
 
-      cy.get('.treemap #rect-sum__SP_POP_TOTL').should(
```

Comment:
These kind of tests should be handled by Applitools or Chromatic. These selectors don't exist anymore now that we're using canvas to draw the charts.